flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -132,9 +134,11 @@ public HiveCatalog(String catalogName, @Nullable String defaultDatabase, @Nullab
 		this.hiveConf = hiveConf == null ? getHiveConf(null) : hiveConf;
```

Comment:
Is there a use case where hiveConf might be null? If not, we can assert it's not null.